src/Icons/doc/index.rst

```diff
@@ -167,9 +167,6 @@ a subdirectory, the *name* will be ``subdirectory:icon_name``.
 Icons On-Demand
 ~~~~~~~~~~~~~~~
 
-`ux.symfony.com/icons`_ has a huge searchable repository of icons
```

Comment:
@smnandre is it the first one or the second one we should keep? the on-demande is different

Reply:
I think in RST it's  `_`  to emphasis

Reply:
the first does not render correctly

https://symfony.com/bundles/ux-icons/current/index.html#icons-on-demand

![image](https://github.com/symfony/ux/assets/48644518/253e190b-109f-40c0-a402-4614cf4ff23c)

Reply:
Yes, the `_ ... _` is italics in Markdown. In RST, italics is `* ... *` and bold is `** ... **`

But here the problem is that this phrase is wrongly repeated, so let's remove it entirely.